java/integTest/src/test/java/glide/TestConfiguration.java

```diff
@@ -6,9 +6,9 @@
 
 public final class TestConfiguration {
     // All redis servers are hosted on localhost
-    public static final int[] STANDALONE_PORTS = getPortsFromProperty("test.redis.standalone.ports");
-    public static final int[] CLUSTER_PORTS = getPortsFromProperty("test.redis.cluster.ports");
-    public static final Semver REDIS_VERSION = new Semver(System.getProperty("test.redis.version"));
+    public static final int[] STANDALONE_PORTS = getPortsFromProperty("test.server.standalone.ports");
```

Comment:
are these properties passed in from CI anywhere where we would also need to update to `test.server.standalone.*`?

Reply:
Oh I forgot to add `build.gradle` 🤦